app.rb

```diff
@@ -15,6 +15,72 @@ def list_persons(persons)
     end
   end
 
+  def add_persons
+    puts "Do you want to create \n1. a student or \n2. a teacher? [Inpt the number]"
+    case gets.chomp
+    when '1'
+      person_student
+    when '2'
+      person_teacher
+    end
+    ask
+  end
+
+  def person_student
+    puts 'Enter the name of the student'
+    name = gets.chomp
+    puts 'Enter the age of the student'
+    age = gets.chomp
+    classroom = 'Math'
+    new_student = Student.new(classroom, age, name)
+    create_person(@person, new_student)
+    puts "Student #{name} has been added to the library"
```

Comment:
- [ ] In all of your methods in the `App.rb` file, you're both getting `an input from users` and then `creating an object` using that input. This method does not follow the single responsibility principle as a method should only do 1, ask for an input from the user or create an object. Please separate these operations into different methods to achieve the `single responsibility principle`. You may have a separate class for validating and getting inputs if you want. Kindly do this for all your methods in App.rb.